tests/Stuart/tests/ClientTest.php

```diff
@@ -144,4 +144,24 @@ public function test_validate_a_dropoff_address()
         self::assertEquals($validity->success, true);
         \Phake::verify($this->httpClient)->performGet('/v2/addresses/validate', $query);
     }
+
+    public function test_get_scheduling_slots()
+    {
+        \Phake::when($this->httpClient)->performGet(\Phake::anyParameters())->thenReturn(
+            new ApiResponse(200, $this->mock->scheduling_slots_response_json())
+        );
+
+        $city = 'London';
```

Comment:
the city must be passed in lowercase, it might be confusing for other integrators (see [documentation](https://stuart.api-docs.io/v2/jobs/get-scheduling-slots)). Should be `london`.